watertap/examples/flowsheets/full_treatment_train/flowsheet_components/chemistry/tests/test_posttreatment.py

```diff
@@ -44,7 +44,7 @@
 def test_ideal_naocl_mixer():
     model = run_ideal_naocl_mixer_example()
     assert model.fs.ideal_naocl_mixer_unit.dosing_rate.value == pytest.approx(
-        0.4, rel=1e-3
+        0.4e-6, rel=1e-3
```

Comment:
Were the units changed here, explaining the change in order of magnitude?

Reply:
(looks like ppm representation converted to mass fraction)

Reply:
Yes, here: https://github.com/watertap-org/watertap/pull/496/files#r855550900